serializer/custom_encoders.rst

```diff
@@ -42,12 +42,18 @@ create your own encoder that uses the
             return Yaml::parse($data);
         }
 
-        public function supportsDecoding($format, array $context = [])
+        public function supportsDecoding($format)
         {
             return 'yaml' === $format;
         }
     }
 
+.. tip::
+
+    If you need access to ``$context`` in your ``supportsDecoding`` or ``supportsEncoding`` method, make sure
+    to implements ``Symfony\Component\Serializer\Encoder\ContextAwareEncoderInterface`` or ``Symfony\Component\Serializer\Encoder\ContextAwareEncoderInterface`` interface
```

Comment:
implement...

Why the same interface twice?

and a . (Dot) is missing at the end

Reply:
Ah I found out by myself, `Decoder` and `Encoder`....